app/views/sessions/index.html.erb

```diff
@@ -17,7 +17,7 @@
             }
           ) %>
         <% end %>
-      <% elsif params[:starts_at].present? %>
+      <% elsif params[:starts_at].present? || active_session_filters? %>
```

Comment:
I suggest making `filter_params` a [helper_method](https://api.rubyonrails.org/classes/AbstractController/Helpers/ClassMethods.html#method-i-helper_method) and using `filter_params.present?`. This same change should be applied to schedules#show